build/ci/release.yml

```diff
@@ -331,14 +331,26 @@ tasks:
           tool_name: ${tool_name}
       - command: archive.targz_pack
         params:
-          target: src/github.com/mongodb/mongodb-atlas-cli/dist/${tool_name}-deb.tgz
-          source_dir: src/github.com/mongodb/mongodb-atlas-cli/dist/apt
+          target: src/github.com/mongodb/mongodb-atlas-cli/dist/${tool_name}-deb-x86_64.tgz
```

Comment:
split targz files per arch to confirm if some errors we are seeing are based on not able to parse both files